configure.ac

```diff
@@ -210,7 +210,7 @@ AC_DEFUN([CH_CHECK_VERSION], [
 AC_PROG_CC
 
 # Set up CFLAGS.
-ch_cflags='-std=c99 -Wpedantic -Wall'
```

Comment:
Why remove `-Wpendantic`? IIRC you were pretty against removing it in the past.

Reply:
Yeah I know. I wish I remembered why. 🙃